spec/rails_helper.rb

```diff
@@ -54,4 +92,35 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include FactoryGirl::Syntax::Methods
+  config.include ApplicationHelper, type: :view
+  config.include Warden::Test::Helpers, type: :feature
+
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.before(:each) do |example|
+    # Pass `:clean' to destroy objects in fedora/solr and start from scratch
+    ActiveFedora::Cleaner.clean! if example.metadata[:clean]
+
+    if example.metadata[:type] == :feature && Capybara.current_driver != :rack_test
```

Comment:
According to the README you only need to explicitly call start on some strategies. The truncation strategy doesn't need .start. 

If I remove that line with the truncation strategy I get database errors for duplicate records.